packages/skeleton-svelte/src/lib/components/Rating/types.ts

```diff
@@ -1,70 +1,15 @@
+import * as rating from '@zag-js/rating-group';
 import type { Snippet } from 'svelte';
 
-export interface RatingProps {
-	/** Sets the rating value. */
-	value?: number;
-	/** Sets the maximum rating value. */
-	max?: number;
-	/** Sets interactive mode. */
-	interactive?: boolean;
-	/** Sets the rating fractional granularity. */
-	step?: number;
-
-	// Root ---
-	/** Sets base styles. */
+export interface RatingProps extends Omit<rating.Context, 'id'> {
```

Comment:
Just a reminder to implement the comments and JSDocs documentation for both framework `types.ts` files.